Expensify/expensify-common
# Fix CopyText propTypes so it can be imported into other files.

@Gonals @changled  will you please review this?

We cannot access `window.PropTypes` unless it's in the global scope so this library is not currently usable in Concierge.

### Fixed Issues (Comment)
$ https://github.com/Expensify/Web-Expensify/pull/27184#issuecomment-606758379

# Tests / QA
1. Test with https://github.com/Expensify/Web-Expensify/pull/27184
1. Navigate to the Expenses page and click on a transaction id.
1. Verify it is copied to the clipboard.
